packages/amino/src/secp256k1hdwallet.ts

```diff
@@ -107,6 +107,12 @@ interface DerivationInfo {
   readonly prefix: string;
 }
 
+export interface Secp256k1HdWalletOptions {
+  readonly bip39Password?: string;
+  readonly hdPath?: HdPath;
```

Comment:
Now that users have to specify the argument name, we could simplify this to just `path`. But I don't have a strong opinion on this one.

Reply:
We use `hdPath` in a bunch of other places. I think it could get confusing.